src/main/java/io/buildpacks/example/sample/CalculatorService.java

```diff
@@ -1,26 +0,0 @@
-package io.buildpacks.example.sample;
-
-import org.springframework.stereotype.Service;
-
-@Service
```

Comment:
5. **Exception Handling**: Consider using more specific exceptions instead of a generic `IllegalArgumentException` if possible. For example, you could create a custom exception class for division by zero scenarios.